open_event/static/js/admin/event/new.js

```diff
@@ -0,0 +1,81 @@
+$( document ).ready(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2651484784/issues/source?bid=3233864&fileBranchId=3324174#l1)